pkg/util/subnet/generate.go

```diff
@@ -1,7 +0,0 @@
-package subnet
-
-// Copyright (c) Microsoft Corporation.
-// Licensed under the Apache License 2.0.
-
-//go:generate go run ../../../vendor/github.com/golang/mock/mockgen -destination=../mocks/$GOPACKAGE/$GOPACKAGE.go github.com/Azure/ARO-RP/pkg/util/$GOPACKAGE Manager
```

Comment:
it happens to be in disks.go because it's alphabetically the first non _addons.go.  Not a good reason ;-).  Yeah, happy to revert this change and if you want to pull them all out into generate.go files, do.

Reply:
Created https://github.com/Azure/ARO-RP/issues/71